ports/faiss/portfile.cmake

```
@@ -0,0 +1,32 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO facebookresearch/faiss
+    REF ef6e53f8ba6b719a053bee513d3b60151925eef2

```

```suggestion
    REF a17a631dc326b3b394f4e9fb63d0a7af475534dc # v1.6.3
```
I noticed that the latest released version is v1.6.3. So It would be better to use the released version.